lookbook/previews/op_turbo/stream_component_preview.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module OpTurbo
+  # @logical_path OpenProject/OpTurbo
+  class StreamComponentPreview < Lookbook::Preview
+    # Renders a turbo-stream tag with given action and target
+    # @param action select { choices: [append, prepend, replace, update, remove, before, after] }
```

Comment:
🚧  _Rails overrides the `action:` local var in Lookbook preview, one way would be to rename the variable but doesn't seem like a good enough reason._ Thoughts / Ideas? 🤔

Reply:
How about:

```ruby
    # Renders a turbo-stream tag with given action and target
    # @param _action select { choices: [append, prepend, replace, update, remove, before, after] }
    # @param target text
    def default(_action: 'append', target: 'model_id')
      template = template_example_from_action(_action, target)
      render_with_template(locals: { template:, action: _action, target: })
    end
```

![Preview demo](https://github.com/opf/openproject/assets/61627014/7a7d3516-5385-4a5a-9b99-d2f1e543ac2c)

Reply:
Worked beautifully, cheers Aaron ⭐